doc/api/buffer.markdown

```diff
@@ -670,9 +670,19 @@ console.log(buf.toString('ascii'));
   // Prints: Node.js
 ```
 
-### buf.compare(otherBuffer)
+### buf.compare(otherBuffer[, otherStart[, otherEnd[, thisStart[, thisEnd]]]])
 
 * `otherBuffer` {Buffer}
+* `otherStart` {Integer} The offset within `otherBuffer` at which to begin
+  comparison. default = `0`.
+* `otherEnd` {Integer} The offset with `otherBuffer` at which to end comparison.
+  Ignored with `otherStart` is undefined.
+  default = `otherBuffer.byteLength`.
+* `thisStart` {Integer} The offset within `buf` at which to begin comparison.
+  Ignored when `otherStart` is undefined. default = `0`
+* `thisEnd` {Integer} The offset within `buf` at which to end comparison.
```

Comment:
we could but `source`/`target` make a lot more sense in `copy` than they do in `compare`.